README.md

```diff
@@ -1,200 +1,74 @@
 <div align="center">
   <h1>Nhost CLI</h1>
-  <a href="https://docs.nhost.io/cli">Website</a>
-    <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://nhost.io/blog">Blog</a>
-  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="docs/">Command Docs</a>
-  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://discord.com/invite/9V7Qb2U">Support</a>
-  <br />
-  <br />
 </div>
 
 <div align="center">
 
-Launch Blazingly Fast Development Environments For Your Nhost Stack
-
-[![Build](https://github.com/nhost/cli/actions/workflows/build.yaml/badge.svg)](https://github.com/nhost/cli/actions/workflows/build.yaml)
 [![Release](https://github.com/nhost/cli/actions/workflows/release.yaml/badge.svg)](https://github.com/nhost/cli/actions/workflows/release.yaml)
 [![Go Report Card](https://goreportcard.com/badge/github.com/nhost/cli)](https://goreportcard.com/report/github.com/nhost/cli)
-  <a href="https://twitter.com/nhostio" target="_blank" rel="noopener noreferrer">
-      <img src="https://img.shields.io/twitter/follow/nhostio?style=social" />
-    </a>
-
-  <br />
+<a href="https://twitter.com/nhost" target="_blank" rel="noopener noreferrer">
+<img src="https://img.shields.io/twitter/follow/nhost?style=social" />
+</a>
 
 </div>
 
-# Contents
-
-- [Installation](#installation)
-  * [Installing On Windows](#installing-on-windows)
-  * [OS & Platform Support](#os--platform-support)
-- [Getting Started](#getting-started)
-- [Front-end Support](#frontend-support)
-- [Usage](#usage)
-  * [Blank Local app](#blank-local-app)
-  * [Existing Remote app](#existing-remote-app)
-  * [Environment Variables](#environment-variables)
-  * [Debugging](#debugging)
-- [Serverless Functions](#functions)
-- [Migrate To v2](#migration)
-- [Dependencies](#dependencies)
-- [Community](#community)
+[Nhost](http://nhost.io) is an open source Firebase alternative with GraphQL.
 
-## Documentation
+The Nhost CLI is used to get a local development environment. The local development environment will automatically track database migrations and hasura metadata.
 
-- [Advanced Usage](https://github.com/nhost/cli/wiki)
-- [Command Docs](/docs)
-- [Changelog](CHANGELOG.md)
-- [Contributing](CONTRIBUTING.md)
-- [Conduct](CODE_OF_CONDUCT.md)
+It's recommended to use the Nhost CLI and the [Nhost GitHub Integration](https://docs.nhost.io/platform/github-integration) to develop locally and automatically deploy changes to production with a git-based workflow (similar to Netlify & Vercel).
 
-<br>
+## Services:
 
-# Installation
+- [Postgres Database](https://www.postgresql.org/)
```

Comment:
shall we mention here https://github.com/nhost/postgres ?

Reply:
I think let's skip for now until we've released k8s Postgres